frontend/src/components/containers/ContainersDetailContent.js

```diff
@@ -32,8 +32,9 @@ const ContainersDetailContent = ({containersByID, samplesByID, usersByID, get, l
   if (!isLoaded)
     get(id);
 
-  if (isLoaded && !container.parents)
+  if (isLoaded && !container.parents) {
     listParents(id);
+  }
```

Comment:
Inconsistent style, if you're going to add braces to single-line ifs add them everywhere (eg above).